.github/workflows/run_test.yml

```diff
@@ -144,6 +144,13 @@ jobs:
                   command: 'npm install'
                   working-directory: ${{ matrix.test.path }}
 
+            - name: Install experimental dependencies for experimental tests
```

Comment:
Explain in the name that this installation is only for the stable API tests running in experimental mode

Reply:
done